test/config.ts

```diff
@@ -0,0 +1,3 @@
+import loglevel from 'loglevel';
+
+process.env.TRACE_LOG && loglevel.enableAll();
```

Comment:
Is `loglevel.enableAll()` equivalent to setting the most verbose log level? If so, why do we need to set `process.env.TRACE_LOG` instead of just changing the log level?

Reply:
the implemented loglevel override does not seem to work in tests.

Reply:
Created an issue to tackle the configuration mess: https://rsklabs.atlassian.net/browse/PP-318